src/styles/style.scss

```diff
@@ -0,0 +1,52 @@
+/* #region style.scss */
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $w-calendar;
+  margin: 0 auto;
+
+  $days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
```

Comment:
- still not fixed 
<img width="1028" alt="Screenshot 2023-09-27 at 16 11 03" src="https://github.com/mate-academy/layout_calendar/assets/64746943/2e2a0c7e-f1fa-494f-8918-acb9388e6d70">